deployments/atd_service_bot_intake.yaml

```diff
@@ -0,0 +1,61 @@
+###
+### A complete description of a Prefect Deployment for flow 'Service Bot: Intake Issues'
+###
+name: 'Service Bot: Intake Issues'
+description: 'Repo: https://github.com/cityofaustin/atd-service-bot, Sends new issue
+  data from our service portal (knack) to our Github'
+version: fd57a6a658aa9cfb2bd9f5a03b1e36dd
+# The work queue that will handle this deployment's runs
+work_queue_name: atd-service-bot
+work_pool_name: atd-data-03
+tags: []
+parameters: {}
+schedule:
+  cron: '*/3 * * * *'
+  timezone: null
+  day_or: true
+is_schedule_active: false
```

Comment:
I switched this to off because I didn't want the flow to immediately schedule when I applied the deployment.

Reply:
I'm still trying to decide if I just dont apply the deployment until after its merged? I think thats how I'm leaning. but not strongly.

Reply:
Yep, that is what I'm wondering too. I was thinking about having staging deployments and production deployments. A staging deployment would be the starting point for the next developer to make changes and test around. 

Maybe, then the CI could look at staging deployment files on merge to `main`, and, if there was a change, apply the production deployment which always points at `main`. Definitely something that we could talk about in the meeting tomorrow.